Add note about not supported `text-wrap-style.avoid-orphans` (#37460)
mdn/content: files/en-us/web/css/text-wrap-style/index.md

- `stable`
  - : Text is wrapped such that when the user is editing the content, the lines that come before the lines they are editing remain static rather than the whole block of text re-wrapping.

> [!NOTE]

> > The [CSS text](/en-US/docs/Web/CSS/CSS_text) module defines a `avoid-orphans` value for the `text-wrap-style` property to avoid excessively short last lines and expect user agent to consider more than one line when making break decisions, however, this is not supported in any browsers.

Reviewer: ```suggestion
> The [CSS text](/en-US/docs/Web/CSS/CSS_text) module defines an `avoid-orphans` value for the `text-wrap-style` property to avoid excessively short last lines and expect the user agent to consider more than one line when making break decisions. This value is not yet supported in any browser.
```
